analyzer/command/apk/apk.go

```diff
@@ -59,6 +59,10 @@ type pkg struct {
 
 type version map[string]int
 
+func (a alpineCmdAnalyzer) CacheType() types.CacheType {
+	return types.BuiltInCache
+}
+
```

Comment:
Embedding struct looks better since all built-in analyzers should have the same behavior.
https://go.dev/doc/effective_go#embedding